### DISCUSSION TITLE
How to use swin unetr with other dataset

### DISCUSSION THREAD

**Discussion by PathSally** [Project-MONAI/MONAILabel#1520] [Q&A]
Hi there!
I want to build another classification based on swin unetr in model zoo with my own dataset and run it on 3D Slicer. But I don't know where to start now. Can you give me some advice?

**Comment by diazandr3s** [comment]
Hi @PathSally,

Do you mean segmentation or classification? MONAI Label was designed for segmentation tasks.
If it is segmentation, you could import the Swin UNETR network [here](https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L18) in the configs file and then define the arguments here: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L82-L90 

Also, you may find this video useful: https://www.youtube.com/watch?v=3HTh2dqZqew

Hope this helps,
